pkg/autoupgrade/daemon.go

```diff
@@ -222,6 +224,17 @@ func (d *daemon) refreshImages(ctx context.Context, apps map[kclient.ObjectKey]v
 			}
 
 			if updated || strings.TrimPrefix(app.Status.AppImage.Digest, "sha256:") != strings.TrimPrefix(digest, "sha256:") {
+				if !updated && digest != "" {
+					if err := d.client.checkImageAllowed(ctx, app.Namespace, nextAppImage); err != nil {
+						if errors.Is(err, &imageallowrules.ErrImageNotAllowed{}) {
+							logrus.Warnf("Updated image %s for %s/%s is not allowed: %v", nextAppImage, app.Namespace, app.Name, err)
```

Comment:
i think this should be info level, not warn. To me, warn indicates something is wrong with the app. In this case, nothing wrong, simply the image doesnt match criteria.

Reply:
Potentially even Debug level, just forgot to change it.
Info level could already be too noisy.
```suggestion
							logrus.Debugf("Updated image %s for %s/%s is not allowed: %v", nextAppImage, app.Namespace, app.Name, err)
```